examples/gallery/lines/roads.py

```diff
@@ -24,10 +24,10 @@
 
 fig = pygmt.Figure()
 
-# Define target region around O'ahu (Hawai'i)
+# Define target region around Oʻahu (Hawaiʻi)
 region = [-158.3, -157.6, 21.2, 21.75]  # xmin, xmax, ymin, ymax
 
-title = r"Main roads of O\047ahu (Hawai\047i)"  # \047 is octal code for '
+title = r"Main roads of O\140ahu (Hawai\140i)"  # \140 is octal code for `, approximating the Okina ʻ
```

Comment:
Technically \140 is a [backtick](https://en.wikipedia.org/wiki/Backtick) ` (U+0060), but this is the closest approximation I know to the Okina letter `ʻ` (U+02BB) as mentioned at https://github.com/GenericMappingTools/pygmt/pull/2641/files#r1306294445. Anyone who knows how to plot the Okina properly is welcome to chime in!